pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -1307,8 +1307,8 @@ final: prev:
     src = fetchFromGitHub {
       owner = "p00f";
       repo = "clangd_extensions.nvim";
-      rev = "b41da6b0838bd07d1d241f6bc09d054dc5f551fd";
-      sha256 = "04q642hjifrnhffb6vzg2km2vvw3fvghr9rh7l9s37ja4m3g2n3v";
+      rev = "4a822fca87c40099a4e8b27f396e7b24114b60d0";
+      sha256 = "1jw9hrb4577vn6yx037bd7diklcpkajasw7mv70mdwlxckdvvn02";
```

Comment:
Can the scripts be updated to use SRI hashes?

Not for now, but in a future PR.

Reply:
IDK

I only run the script